packages/react/src/experimental/OneDataCollection/Filters/index.tsx

```diff
@@ -113,6 +113,10 @@ const FiltersRoot = <Definition extends FiltersDefinition>({
 
   const [localFiltersValue, setLocalFiltersValue] = useState(filters)
 
+  useEffect(() => {
```

Comment:
be careful with this effect, maybe worth checking if the value has really changed